static/website/js/thread-user.js

```diff
@@ -1,327 +1,71 @@
-    $question_tutorial = $('#id_tutorial');
-    $question_minute_range = $('#id_minute_range');
-    $question_second_range = $('#id_second_range');
-    
-    $question_details_ok.click(function() {
-        $saving.show();
-        var category = $question_category.val();
-        var tutorial = $question_tutorial.val();
-        var minute_range = $question_minute_range.val();
-        var second_range = $question_second_range.val();
-        
-        $.ajax({
-            url: "/ajax-details-update/",
-            data: {
-                qid: $(this).data("qid"),
-                category: category,
-                tutorial: tutorial,
-                minute_range: minute_range,
-                second_range: second_range
-            },
-            type: "POST",
-            success: function(data){
-                if(category != 'None') {
-                    $(".category small a").html(category);
-                    $(".category").show()
-                } else {
-                    $(".category").hide()
-                }
-                if(tutorial!= 'None') {
-                    $(".tutorial small a").html(tutorial);
-                    $(".tutorial").show()
-                } else {
-                    $(".tutorial").hide()
-                }
-                if(minute_range!= 'None') {
-                    $(".minute_range small a").html(minute_range);
-                    $(".minute_range").show()
-                } else {
-                    $(".minute_range").hide()
-                }
-                if(second_range != 'None') {
-                    $(".second_range small a").html(second_range);
-                    $(".second_range").show()
-                } else {
-                    $(".second_range").hide()
-                }
-                $saving.hide();
-                $saved.show();
-                $saved.fadeOut("slow");
-            }
-        });
-    });
-
     /*
      * answer edit section
      * set the dom variables
     */
     $answer_edit = $('.answer .edit');
     $answer_save = $(".answer .save");
-    $answerPanelWrapper = $("#answerPanelWrapper");
-
-    var answerNicEditor = new nicEditor({
-        fullPanel : true,
-        buttonList : ['fontSize','bold','italic','underline','strikeThrough','subscript','superscript','html','image', 'link', 'forecolor', 'bgcolor'],
-        iconsPath: "/static/website/js/nicEditorIcons.gif",
-    });
-    answerNicEditor.panelInstance('answerNicPanel');
 
     $answer_edit.click(function(e) {
         var target = $(this).data("target");
-        answerNicEditor.addInstance(target);
-        $(this).parents("div.answer").prepend($answerPanelWrapper);
-        $answerPanelWrapper.show();
-        $('#answerPanelWrapper .nicEdit-panelContain').parent().width('100%');
-        $('#answerPanelWrapper .nicEdit-panelContain').parent().next().width('100%');
+        var id = $(this).attr("id");
+        $("#"+id+"1").show();
+        $("#"+id+"2").hide();
```

Comment:
I am some what not convinced with these lines of code.
@prashantsinalkar kindly look into this JS

Reply:
Its for the edit and the ck-editor wrapper for the edit answer for multiple answers.

Reply:
its for niceditor.